app/Http/Services/Tabbar/TabbarService.php

```diff
@@ -114,14 +122,15 @@ private function _getDevTabInfo($devParam, $devTabUrlInfo) {
     }
 
     /**
-     * Docsタブの情報を取得する
+     * Docsタブの情報を取得する.
      * 
-     * @param string $docsParam リクエストパラメータのdocsの値
-     * @param array $docsTabUrlInfo Docsタブ内に表示するページのURL一覧
+     * @param string $docsParam      リクエストパラメータのdocsの値
+     * @param array  $docsTabUrlInfo Docsタブ内に表示するページのURL一覧
+     *
      * @return string Docsタブに表示するページのURL
-     * 
      */
-    private function _getDocsTabInfo($docsParam, $docsTabUrlInfo) {
+    private function _getDocsTabInfo($docsParam, $docsTabUrlInfo)
```

Comment:
__[code_sniffer]__
* Method name "_getDocsTabInfo" should not be prefixed with an underscore to indicate visibility